apps/frontend/src/pages/content.css

```diff
@@ -0,0 +1,7 @@
+.toc-level {
```

Comment:
Can this be set as [Nested Selectors](https://emotion.sh/docs/nested) in the component lib / rich text?

Reply:
Sure :+1: